client/components/MessageTypes/away.vue

```diff
@@ -23,5 +25,10 @@ export default {
 		network: Object,
 		message: Object,
 	},
+	computed: {
+		awayMessage() {
+			return this.message.text.trim();
```

Comment:
this is not really necessary to shove in this PR, but we shouldn't show empty away messages, as it looks like `( )`